src/constants/cookie-categories.ts

```diff
@@ -0,0 +1,6 @@
+export enum CookieCategories {
```

Comment:
This enum isn't used anywhere... this is weird?

Reply:
Ahh yeah, I had some additional work on this branch that used this enum, but after syncing with Mitch, decided to cut it. I'll just drop this enum for now 👍